src/components/LeftSidebar/NewGroupConversation/NewGroupConversation.vue

```diff
@@ -456,12 +456,13 @@ export default {
 	the margin applied to the content is added to the total modal width,
 	so here we subtract it to the width and height of the content.
 	*/
-	height: 100%;
+	height: auto;
 	padding: 20px;
 	display: flex;
 	flex-direction: column;
 	justify-content: space-between;
 	position: relative;
+	gap: 10px;
```

Comment:
It's only the gap between header and content. Do we need that large than now?
If yes, maybe increase margin-bottom instead?

Reply:
I forgot to remove it because the navigation was inside  `new-group-conversation`